### ISSUE TITLE
junit output invalid when there are no errors

### ISSUE THREAD

**Issue by FISHMANPET** [DavidAnson/markdownlint-cli2#123]
When there are no errors, the junit formatter reports a single non-failing test case.  The output will look like this:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="markdownlint-cli2-formatter-junit" time="0" tests="1" failures="0" errors="0" skipped="0">
    <testcase time="0"/>
  </testsuite>
</testsuites>
```

However, Azure DevOps isn't able to process this, with the error message
> ##[warning]Few results could not be published - No name attribute was found for some of the test cases

The single successful test case should have some name, so that it can be properly parsed.

I'm working a PR for this, right now I'm just going to give it the same name as the test suite, which is
```js
path.basename(__filename).replace(/\.js$/u, "")
```

I'll see if there's any other properties that are need as well and add them in.

**Comment by FISHMANPET** [comment]
In addition to the warning printed, the Azure DevOps test explorer doesn't show any tests when the junit file with no testcase name is uploaded
![image](https://user-images.githubusercontent.com/7810151/219084930-b8aac6fa-c4c1-4c59-989e-db54c3a50e33.png)

With my change, Azure DevOps is able to see the one test result:
> Publishing test results to test run '7751'.
> TestResults To Publish 1, Test run id:7751
> Test results publishing 1, remaining: 0. Test run id: 7751

And the test explorer shows tests (your able to click through this to see the single test case)
![image](https://user-images.githubusercontent.com/7810151/219085576-a81d8b8d-3520-43ff-94e8-ddad6b59c43c.png)

**Comment by DavidAnson** [comment]
Fixed in latest published version `0.0.6` (and possibly the previous).
